pkg/controller/queuejob/queuejob_controller_ex.go

```diff
@@ -709,10 +748,30 @@ func (cc *XController) updateQueueJob(oldObj, newObj interface{}) {
 		glog.Errorf("newObj is not AppWrapper")
 		return
 	}
-	glog.V(10).Infof("[TTime] %s, %s: updateQueueJob delay: %s", time.Now().String(), newQJ.Name, time.Now().Sub(newQJ.CreationTimestamp.Time))
+	oldQJ, ok := oldObj.(*arbv1.AppWrapper)
+	if !ok {
+		glog.Errorf("[Informer-updateQJ] oldObj is not AppWrapper.  enqueue(newQJ).  oldObj=%+v", oldObj)
+		glog.V(4).Infof("[Informer-updateQJ] BadOld enqueue %s newQJ=%p newQJ=%+v", newQJ.Name, newQJ, newQJ)
```

Comment:
It can happen that the oldObj is not an AppWrapper, then we enqueue the newQJ object. Changed BadOld should be Bad_oldObj.  Want to keep it one_word to be compared with the "normal" line.  Hope this helps.  
Reduced logging information to address, version and status.  This line of log helps us to know what events are being enqueued, by whom and when.

Reply:
Mods to the messaging look great.  Thanks!